test/e2e/provisioning/deploymentscenario.go

```diff
@@ -34,6 +34,8 @@ func verifyCreateUpdateAndDelete(kubeConfig, manifestPath string, parameters []s
 	if err != nil {
 		return err
 	}
+	ctx := context.Background()
```

Comment:
Hacked it a bit here but if I introducing context at a higher level requires a lot of refactoring since our e2e tests just use new context everywhere.